backend/db-setup/anonymize-db.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+echo "Cleaning up the directory for pg dump."
+sudo mkdir -p backend/db-setup/snapshots/;
```

Comment:
Any chance you can add in an additional `echo` statement here that lets the user know you're prompting for `sudo` access? I spent a little too much time trying to dig up database passwords before I realized this was a `sudo` request. 😅

Reply:
done